src/redis-benchmark.c

```diff
@@ -1614,8 +1614,8 @@ int parseOptions(int argc, char **argv) {
 " -a <password>      Password for Redis Auth\n"
 " --user <username>  Used to send ACL style 'AUTH username pass'. Needs -a.\n"
 " -u <uri>           Server URI.\n"
-" -c <clients>       Number of parallel connections (default 50). Note: number of\n"
-"                    clients has to be the same or higher than the amount of nodes.\n"
+" -c <clients>       Number of parallel connections (default 50). Note: If --cluster is used\n
+"                    then number of clients has to be the same or higher than the number of nodes.\n"
```

Comment:
Good. Only now the output lines are a bit long. The old output was not more than 80 columns. It is the default size of a terminal. I think we can follow this style.

```suggestion
" -c <clients>       Number of parallel connections (default 50).\n"
"                    Note: If --cluster is used, then number of clients has to be\n"
"                    the same or higher than the number of nodes.\n"
```